benchmark/benchmark_test.go

```diff
@@ -100,3 +100,28 @@ func BenchmarkPureMergeSort(b *testing.B) {
 		evaluator.Eval(program, env)
 	}
 }
+
+func BenchmarkRockHopper(b *testing.B) {
+	program, env := testBuild(`
+	let rock_hopper = fn(arr, pos) { if (pos > len(arr) - 1) { true } else { if (arr[pos] == 0) { false } else { rock_hopper(arr, pos + arr[pos]) }}}
+	`)
+	for i := 0; i < b.N; i++ {
+		evaluator.Eval(program, env)
+	}
+}
+
+func BenchmarkPureRockHopper(b *testing.B) {
```

Comment:
These benchmarks are not actually accelerated yet. They are more just aspirational. Once full support of arrays are in these should be faster I think...